Gruntfile.js

```diff
@@ -157,19 +231,87 @@ module.exports = function(grunt) {
             'ui-common': {
                 src: 'build/test-coverage/lcov/**/*.info',
             }
-        }
+        },
+        // file concatenation
+        concat: {
+            options: {
+                banner: cssFileHeader,
+            },
+            // css for the "page" template
+            pageCss: {
+                src: pageCssFiles,
+                dest: kbaseStaticDir + 'css/page_concat.css',
+                nonull: true,
+            },
+            // a single file containing all the KBase css!
+            allCss: {
+                src: allCssFiles,
+                dest: kbaseStaticDir + 'css/all_concat.css',
+                nonull: true,
+            }
+        },
 
+        // check the source css files are not completely horrendous
+        stylelint: {
+            options: stylelintOptions,
+            kbaseCss: {
+                src: staticFilesNoConcat,
+            },
+        },
+
+        // Run CSS / SCSS-related tasks
+        // these all modify files in place
+        postcss: {
+            // clean up the files in kbase-extension/static/kbase/(css|custom)/*.css
+            kbaseCss: {
+                options: cleanExistingCssOptions,
+                src: staticFilesNoConcat,
+            },
+            // autoprefix and minify the concatenated css files
+            concat: {
+                options: {
+                    processors: [
+                        // add vendor prefixes
+                        require('autoprefixer')(),
+                        // add minification later
+                        // require('cssnano')([
+                        //     "default",
+                        //     {
+                        //         "normalizeWhitespace": {
+                        //             "exclude": true
+                        //         },
+                        //     }
+                        // ]),
```

Comment:
I commented this out for the time being, but once we have verified that the workflow is OK, it'd be best to serve minified files.